WordPress/Classes/ViewRelated/Comments/EditCommentTableViewController.swift

```diff
@@ -122,4 +142,42 @@ private extension EditCommentTableViewController {
         dismiss(animated: true)
     }
 
+    // MARK: - Tap gesture handling
+
+    @objc func dismissKeyboard() {
+        view.endEditing(true)
+    }
+
+    // MARK: - Table sections
+
+    private enum TableSections: Int, CaseIterable {
+        // The case order dictates the table row order.
+        case name
+        case webAddress
+        case emailAddress
+        case comment
+
+        var header: String {
+            switch self {
+            case .name: return NSLocalizedString("Name", comment: "Header for a comment author's name, shown when editing a comment.").localizedUppercase
```

Comment:
minor nit, wanted to clarify: is this intended to be inlined with the case, or should the return statement be on the new line?

Reply:
Woops. Tis fixed. Thank you.